.changeset/orange-jars-relax.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
COuld also remove `hello` folder

Reply:
i still feel it's good to have a hello world test, should we rename it basic?